src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareModal.tsx

```diff
@@ -147,102 +162,139 @@ export const DocShareModal = ({ doc, onClose }: Props) => {
     });
   };
 
+  const getListHeight = () => {
+    const contentHeight = 690;
+    const inputHeight = 70;
+    const marginTop = 11;
+
+    const footerHeight = footerRef.current?.clientHeight ?? 0;
+    const selectedUsersHeight = selectedUsersRef.current?.clientHeight ?? 0;
+
+    return (
+      contentHeight -
+      footerHeight -
+      selectedUsersHeight -
+      inputHeight -
+      marginTop
+    );
+  };
+
+  const handleRef = (node: HTMLDivElement | null) => {
+    if (!node) {
+      return;
+    }
+
+    footerRef.current = node;
+    setListHeight(getListHeight());
+  };
+
   return (
-    <Modal
-      isOpen
-      closeOnClickOutside
-      data-testid="doc-share-modal"
-      aria-label={t('Share modal')}
-      size={isDesktop ? ModalSize.LARGE : ModalSize.FULL}
-      onClose={onClose}
-      title={<Box $align="flex-start">{t('Share the document')}</Box>}
-    >
-      <Box
+    <>
+      <Modal
+        isOpen
+        closeOnClickOutside
+        data-testid="doc-share-modal"
```

Comment:
It is usually better practice to use accessibility props to target an element in your tests, if you have a aria label already set, better to use it; perso I use more `data-testid` as a fallback

Reply:
it was already here before i not that and I do it in another PR